airbrake/utils.py

```diff
@@ -16,6 +17,15 @@
     TypeType = type
 
 
+class FailProofJSONEncoder(json.JSONEncoder):
+    """Uses object's representation for unsupported types."""
+
+    # pylint: disable=E0202
```

Comment:
@argos83 comments like these can typically be more valuable inline. 

Would you be against repharsing your [comment](https://github.com/airbrake/airbrake-python/pull/27#discussion_r54394911) and adding it inline? Or can you think of a reason not to include it inline?

Either way this looks good. Let me know what you think.

Thanks!